### PR TITLE
Add meta tags for cronos docs

### PR DESCRIPTION
**Pull request by cdc-william**
Adding meta tags in cronos docs for better seo performance

image

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "twitter:site"
+    content: "@cryptocom"
+  - name: "twitter:card"
+    content: summary_large_image
+  - name: "twitter:description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "twitter-image"
+    content: https://cronos.crypto.org/og-image.png
+---
```
Comment (ysong42) [inline docs/getting-started/README.md:25]: According to this tutorial: https://www.adamdehaven.com/blog/how-to-add-metadata-canonical-urls-and-structured-data-to-your-vuepress-site/#add-the-canonical-url-to-every-page

We could use `canonicalUrl` in the yaml to specific a canonical link. (If we upgrade vuepress to v1.7.1+)

Such as `canonicalUrl: https://cronos.crypto.org/docs/getting-started/`.

**Review by ysong42 — CHANGES_REQUESTED** [review]
Thanks for updating the meta tags. 🙇 

Some miner issues spotted, which is in the comments.

And I think, we miss the meta tag settings for home page, which could be configured in `docs/README.md`.

Thank you! 🙏

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+    content: https://cronos.crypto.org/og-image.png
+  - name: "twitter:title"
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "twitter:site"
+    content: "@cryptocom"
+  - name: "twitter:card"
+    content: summary_large_image
+  - name: "twitter:description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "twitter-image"
```
[the remark this answers is not in the document]
Comment (cdc-william) [reply docs/getting-started/README.md]: It is fixed now. Thanks!

File: docs/getting-started/README.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "og:title"
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "og:type"
+    content: Website
+  - name: "og:description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "og-image"
```
[the remark this answers is not in the document]
Comment (cdc-william) [reply docs/getting-started/README.md]: It is fixed now. Thanks!

**Comment by ysong42** [comment]
Thanks! Good to merge! Later I will update the package version and add the caconical link~

**Review by ysong42 — APPROVED** [review]

**Pull request merged**